onsite/phosphors_scoring.py

```
@@ -0,0 +1,540 @@
+        hit.setMetaValue("search_engine_sequence", original_seq_str)
+        
+        # Check for phosphorylation sites
+        has_phospho = False
+        is_decoy = False
+        for aa in ['S', 'T', 'Y', 'A']:
+            if f"{aa}(Phospho)" in original_seq_str:
+                has_phospho = True
+                break
+            if f"{aa}(PhosphoDecoy)" in original_seq_str:
+                has_phospho = True
+                is_decoy = True
```

Ruff (0.14.0)

Local variable `is_decoy` is assigned to but never used

Remove assignment to unused variable `is_decoy`

(F841)